test/e2e/cluster.sh

```diff
@@ -42,6 +42,22 @@ ${ADD_NODE_POOL_INPUT}
 END
 fi
 
+if [ -n "$PRIVATE_SSH_KEY_FILE" ]; then
+  PRIVATE_SSH_KEY_FILE=$(realpath --relative-to=$(pwd) ${PRIVATE_SSH_KEY_FILE})
+fi
+
+function tryExit {
+  if [ "${AZURE_ENV}" != "AzureStackCloud" ]; then
+    exit 1
+  fi
+}
+
+function renameResultsFile {
+  if [ "${AZURE_ENV}" == "AzureStackCloud" ] && [ "${SKIP_TEST}" == "false" ]; then
```

Comment:
What about variables `SKIP_TESTS_AFTER_*`? I may be simpler to check if the results file exists